tests/scenarios/pytest/test_upgrade_scenario.py

```diff
@@ -52,6 +53,4 @@ def test_find_latest_version(
         assert type(catched_error) == error_type
     else:
         assert ver == ver_found
-    cast(Mock, app_test_suite.steps.upgrade_test_runner.requests.get).assert_called_once_with(
-        catalog_url + "/index.yaml"
-    )
+    cast(Mock, app_test_suite.steps.scenarios.upgrade.requests.get).assert_called_once_with(catalog_url + "/index.yaml")
```

Comment:
We're using black (https://github.com/psf/black) to avoid exactly this problem. 'black' is opinionated code formatter for python, where there is basically only one way to format code. So even if your editor will change it in a weird way, `pre-commit` will run `black` on commit and formatting will be back to standard.